euth/dashboard/templates/euth_dashboard/project_form.html

```diff
@@ -10,6 +11,10 @@ <h1 class="black-divider dashboard-title dashboard-title-no-margin">{{ heading }
         <div class="errorbar">
             {% trans 'Unable to submit form. Please fix the errors.' %}
         </div>
+        {% elif project and project.is_archived %}
```

Comment:
You don't need to do such checks in django templates. The template language will them do itself.